dash-docs/protocol-documentation.md

```diff
@@ -99,6 +99,9 @@ Every few minutes, masternodes ping the network with a message that propagates t
 | 32 | blockHash | uint256 | Current chaintip blockhash minus 12
 | 8 | sigTime | int64_t | Signature time for this ping
 | 71-73 | vchSig | char[] | Signature of this message by masternode (verifiable via pubKeyMasternode)
+| 1 | fSentinelIsCurrent | bool | true if last sentinel ping was actual
```

Comment:
> true if last sentinel ping was actual
I've wondered about this wording previously -- should this be changed to "... sentinel ping was current" instead?